crossIBC.ps1

```diff
@@ -0,0 +1,62 @@
+param(
```

Comment:
Who is in charge of calling this script? / how is the flow for this going to work?

Reply:
The way that I am envisioning this is that it gets called as a build step in the definition. It requires that we have taken the Linux binary artifacts and have them in a directory at the root of the repo called Linux. I could add a command line argument that would allow that folder to be specified if we want, but since we should be controlling the whole process I had not done it yet.

Reply:
Please do not put this script in the root of the repo, if you need this extra script it should go somewhere under the eng directory. 

cc @ericstj

Reply:
what do you mean by: 
>  It requires that we have taken the Linux binary artifacts and have them in a directory at the root of the repo called Linux

like, we need to modify where the build is currently dropping the Linux assets in order for this to work?

Reply:
We will need to make changes once we are ready to start doing the cross application. Specifically, we will need to have a step take the built Linux artifacts and move them over to a Windows machine where we can run this script to apply the data then move the Linux artifacts back.